#Service to open and close the hand with delays
float64 delay1      # Seconds to wait before setting angle1
float64 angle1    # Nominal position 1 of the motor
float64 delay2      # Seconds to wait before setting angle2
float64 angle2    # Nominal position 2 of the motor
---
int64 ret         # Set to 1 (success) or 0 (error)
string msg        # Error description
